src/Format.cpp

```diff
@@ -177,7 +160,7 @@ std::vector<char> format::VideoBufferToBMP(const VideoBuffer & vidBuf)
 
 	data.insert(data.end(), buffer, buffer+54);
 
-	char *currentRow = (char *)malloc((vidBuf.Width * 3) + padding);
+	auto currentRow = std::vector<char>(vidBuf.Width * 3);
```

Comment:
Not a large fan of `auto`. Why not `std::vector<char> currentRow(vidBuf.Width * 3)`?